classes/Snackbar.js

```diff
@@ -137,21 +135,17 @@ class Snackbar {
               <div id="buttons" class="flex justify-center items-center">
               </div>
             </div>`;
-    this.id = `tawilwind-snackbar-${
-      numbers[Math.floor(Math.random() * Math.floor(11))]
-    }`;
+    this.id = `tawilwind-snackbar-${numbers[Math.floor(Math.random() * Math.floor(11))]
+      }`;
     wrapper.id = this.id;
     let buttonWrapper = wrapper
       .querySelector(".twsnackbar")
       .querySelector("#buttons");
     this.buttons.forEach((button) => {
       let newButton = document.createElement("DIV");
-      newButton.classList = `cursor-pointer hover:bg-${this.color}-${
-        parseInt(this.tone) + 100
-      } p-2 rounded flex justify-center items-center`;
```

Comment:
```
hover:bg-${this.color}-${
        parseInt(this.tone) + 100
      }
```
This is the only design spec that I had to leave out. Let me know if we would want to keep this, so I'll make the changes accordingly.